compiler/plc_project/src/project.rs

```diff
@@ -309,8 +309,11 @@ fn resolve_file_paths(location: Option<&Path>, inputs: Vec<PathBuf>) -> Result<V
     for input in &inputs {
         let input = location.map(|it| it.join(input)).unwrap_or(input.to_path_buf());
         let path = &input.to_string_lossy();
-        let paths = glob(path).context(format!("Failed to read glob pattern {path}"))?;
+        if !input.exists() {
```

Comment:
I didn't get the chance to test this yet, but since the check is happening before the globbing, would this work when we pass a `*.st` or a similar glob?
Keep in mind that your shell might resolve the globs before you get to this line, so verify the following:
`plc "*.st" -o test.o`  the quotes will force your shell to ignore the globs so that they are resolved by the compiler.